eng/Build.props

```diff
@@ -127,6 +127,7 @@
         -->
         <DotNetProjects Include="
                           $(RepoRoot)src\Framework\ref\Microsoft.AspNetCore.App.Ref.csproj;
+                          $(RepoRoot)src\Framework\ref\Microsoft.AspNetCore.App.Ref.Internal.csproj;
```

Comment:
Recommend moving this project elsewhere. As-is it's treated as a ref/ project and won't be packed.

It's possible having two projects in one directory also explains why the PR validation build didn't produce an App.Ref package.

Reply:
Validation didn't produce the package because 3.1 isn't set up to produce a ref pack right now - but probably a good idea to separate this into another folder